galpy/potential/MovingObjectPotential.py

```diff
@@ -67,17 +49,34 @@ def __init__(self,orbit,amp=1.,GM=.06,
            2011-04-10 - Started - Bovy (NYU)
 
         """
-        Potential.__init__(self,amp=amp*GM,ro=ro,vo=vo,amp_units='mass')
-        if _APY_LOADED and isinstance(softening_length,units.Quantity):
-            softening_length= softening_length.to(units.kpc).value/self._ro
-        # Make sure we aren't getting physical outputs
+
+        Potential.__init__(self,ro=ro,vo=vo)
+        isList = isinstance(pot,list)
```

Comment:
I don't think there is a real need for ``isList``. I would do:
* if pot is None: # setup Plummer potential
* else:
    pot= flatten(pot)
    if _isNonAxi(pot): # error...
    self._pot= pot # No need to copy I think, because it shouldn't get changed anyway